packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -177,25 +209,25 @@ export function translate(
     defaultMessage: '',
   }
 ) {
+  const shouldUsePseudoLocale = isPseudoLocale();
+
   if (!id || !isString(id)) {
     throw new Error('[I18n] An `id` must be a non-empty string to translate a message.');
   }
 
-  const message = getMessageById(id);
+  const message = shouldUsePseudoLocale ? defaultMessage : getMessageById(id);
 
   if (!message && !defaultMessage) {
     throw new Error(`[I18n] Cannot format message: "${id}". Default message must be provided.`);
   }
 
-  if (!hasValues(values)) {
-    return message || defaultMessage;
-  }
-
   if (message) {
     try {
       const msg = getMessageFormat(message, getLocale(), getFormats());
```

Comment:
__optional nit:__ maybe write it like this, to have `format` call just in one place?
```ts
const formattedMessage = getMessageFormat(message, getLocale(), getFormats()).format(values);
return shouldUsePseudoLocale
        ? translateUsingPseudoLocale(formattedMessage)
        : formattedMessage;
```